src/Modules/DisplayTask/index.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { ChangeEvent, useState } from "react";
+import { useNavigate, useParams } from "react-router-dom";
+
+import DisplayTaskComponent from "./components";
+import { useFetch } from "../../CustomHooks/useFetch";
+import Modal from "../../Shared/components/Modal";
+
+import { ROUTES } from "../../appContants";
+import { TASK_STATE } from "../../constant";
+import { TODO_URL } from "../../utils/constant";
+
+const DisplayTaskContainer = () => {
+  const { id } = useParams();
+  const { data, loading, error } = useFetch(
+    TODO_URL + `/${id}`
+  );
+  const [openModal, setOpenModal] = useState<boolean>(false);
+  const navigate = useNavigate();
+
+  const deleteTodo = (todoId: number) => {
+    fetch(`${TODO_URL}/${todoId}`, {
+      method: "Delete",
+      headers: {
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify(data),
+    })
+      .then((res) => res.json())
+      .catch((err) => console.log(err));
+    navigate(ROUTES.HOME);
+  };
+
+  const markTodoCompleted = (e: ChangeEvent<HTMLSelectElement>) => {
+    let tempObj = {
+      ...data,
+      completed: e.target.value === TASK_STATE.COMPLETED ? true : false,
```

Comment:
e.target.value === TASK_STATE.COMPLETED ? true : false,

can be replace by 

e.target.value === TASK_STATE.COMPLETED

Reply:
done